src/cupbearer/scripts/eval_detector.py

```diff
@@ -15,6 +16,11 @@ def main(cfg: Config):
         save_dir=cfg.dir.path,
     )
 
+    if not cfg.no_load and hasattr(cfg.task, "backdoor"):
```

Comment:
I'm a bit worried this could break silently (e.g. on a refactor that changes the `backdoor` attribute). I know I have a few similar checks elsewhere, just trying to figure out if there's a better way here.

Is there any reason the approach I mentioned in #4 wouldn't work? I.e. run this `load` method inside `BackdoorDetection._get_anomalous_test_data()` and use the path that's passed to `BackdoorDetection` anyway. Then `no_load` could be an attribute on `BackdoorDetection` instead, and we wouldn't need the `hasattr` check

Reply:
I'll take a look. I remember looking at that function previously, don't know if there was a reason why I didn't put it there.